exercises/dnd-character/dnd_character_test.py

```diff
@@ -0,0 +1,74 @@
+import unittest
+
+from dnd_character import Character
+
+
+# Tests adapted from 'problem-specifications//canonical-data.json' @ v1.0.0
+
+class DnDCharacterTest(unittest.TestCase):
+    def test_modifier_for_score_3_is_n4(self):
+        self.assertEqual(Character().modifier(3), -4)
```

Comment:
`modifier` should probably be separate from `Character`, as it does not use any of `Character`'s members (i.e. it is pure).

Reply:
Deleted the modifier method and created a modifier function in example.py